skeletonkey/core.py

```diff
@@ -118,10 +120,9 @@ def instantiate(namespace: argparse.Namespace, **kwargs) -> Any:
     Raises:
         TypeError: If the class is missing specific parameters.
     """
-    obj_kwargs = vars(namespace).copy()
-    target_keyword = "_target_"
-    class_obj = import_class(obj_kwargs[target_keyword])
-    del obj_kwargs[target_keyword]
+    obj_kwargs = vars(key).copy()
+    class_obj = import_class(obj_kwargs[TARGET_KEYWORD])
+    del obj_kwargs[TARGET_KEYWORD]

```

Comment:
I am not a fan of the pattern where you use a global variable in the body of a function. Instead it should be used as the default parameter in a function. One future feature I would like to have is the ability to disable skeletonkey features from "unlock", and this pattern would make it hard.

Reply:
Ah, I'd never thought of that! That's a good pattern to stick to.